src/EnergyPlus/SystemAvailabilityManager.cc

```diff
@@ -4471,6 +4486,13 @@ namespace SystemAvailabilityManager {
 				ErrorsFound = true;
 			}
 
+			if ( !lNumericFieldBlanks( 8 ) ) {
```

Comment:
It's just that the \default in the IDD is only informative now instead of active, because the real default is hidden in the code.  I know, lots of objects do this already, but I've been trying to move away from it.  Maybe it will change when the new input processor flow drops in.  Ok to leave as-is.